src/components/molecules/ImageInput/ImageCollectionInput.tsx

```diff
@@ -145,6 +142,7 @@ export const ImageCollectionInput: React.FC<ImageInputProps> = (props) => {
         }}
       >
         <input
+          key={fileInputKey.toString()} // trick react to recreate the file input
```

Comment:
You able to leave a slightly more detailed comment? Trying to understand how forcing re render affects things

Reply:
sure